tests/Unit/Domain/Test_Tags.cpp

```diff
@@ -60,6 +69,48 @@ void test_simple_tags() {
       "Jacobian(ElementLogical,Inertial)");
 }
 
+void test_center_tags() {
+  TestHelpers::db::test_base_tag<Tags::ObjectCenter<ObjectLabel::A>>(
+      "ObjectCenter");
+  TestHelpers::db::test_base_tag<Tags::ObjectCenter<ObjectLabel::B>>(
+      "ObjectCenter");
+  TestHelpers::db::test_simple_tag<Tags::ExcisionCenter<ObjectLabel::A>>(
+      "CenterObjectA");
+  TestHelpers::db::test_simple_tag<Tags::ExcisionCenter<ObjectLabel::B>>(
+      "CenterObjectB");
+
+  using Object = domain::creators::BinaryCompactObject::Object;
+  using Refinement =
+      domain::creators::BinaryCompactObject::InitialRefinement::type;
+  using GridPoints =
+      domain::creators::BinaryCompactObject::InitialGridPoints::type;
+
+  const std::unique_ptr<DomainCreator<3>> domain_creator =
+      std::make_unique<domain::creators::BinaryCompactObject>(
+          Object{0.2, 5.0, 8.0, true, true}, Object{0.6, 4.0, -5.5, true, true},
+          100.0, 500.0, Refinement{1}, GridPoints{5});
```

Comment:
just put `size_t{1}` and `size_t{5}` here, and delete the type aliases

Reply:
Ugh clang-10 tricking me into thinking I don't need `1_st`... 🙄